gitlab/packages/gitlab/gitlab_stats.js

```diff
@@ -0,0 +1,90 @@
+// jshint esversion: 9
+
+/**
+ * @description null
+ * @param {ParamsType} params list of command parameters
+ * @param {?string} commandText text message
+ * @param {!object} [secrets = {}] list of secrets
+ * @return {Promise<SlackBodyType>} Response body
+ */
+
+const axios = require('axios');
+
+async function fetchStats(url) {
+    return await axios.get(url)
+    .then(response => { return response.data; })
+    .catch(error => { return error.response.data; });
+}
+
+async function _command(params, commandText, secrets = {}) {
+  const {
+    repo
+  } = params;
+  
+  if (!secrets.gitlabToken)

```

Comment:
Nit: braces around `if` statement to keep it consistent with other commands.

Reply:
Understood 👍